### PR TITLE
[CELEBORN-1306] Introduce `master` module for dashboard frontend

### PR DESCRIPTION
**Pull request by tiny-dust**
### What changes were proposed in this pull request?

Master list:
image

Master Detail:
 Workers: 
image
 Application:
image
 Configuration:
image
 FlameGraph:
image
 ThreadDump:
image
 Metrics:
image
 Logs:
image
 LogList:
image

### Why are the changes needed?

The Master module provides Celeborn with information about the master.

### Does this PR introduce _any_ user-facing change?

No.

### How was this patch tested?

Local test.

### CODE REVIEW

File: web/src/api/models/master/types/master.ts [hunk 1]
```diff
@@ -15,7 +15,27 @@
...
+  state: string
+  rpcPort: number
+  restPort: number
+  version: string
+}
+
 export interface MasterOverview {
   leaderNum: number
   followerNum: number
+  leader: MasterLeader
```
Comment (SteNicholas) [inline web/src/api/models/master/types/master.ts:30]:
```suggestion
  leader: Master
```
Meanwhile, remove interface `MasterLeader`.

**Review by SteNicholas — COMMENTED** [review]
@tiny-dust, LGTM overall. I left the minor comments for this pull request. PTAL.

File: web/src/api/models/master/types/master.ts (hunk 1 above)
Comment (tiny-dust) [reply web/src/api/models/master/types/master.ts]: ok.

**Review by SteNicholas — APPROVED** [review]

**Comment by SteNicholas** [comment]
LGTM. Merging to main(v0.5.0).

**Pull request closed without merging**